File: 01_Language/01_Functions/python/floatval.py
# coding: utf-8

from doubleval import doubleval


def floatval(s):
    return doubleval(s)


if __name__ == '__main__':
    print(floatval("122.34343The"))
    print(floatval("+122.34343."))
    print(floatval("-122.34343.1"))
    print(floatval("-.34343.1"))
    print(floatval("The122.34343"))

File: 01_Language/01_Functions/js/floatval.js
function floatval(v) {
    return parseFloat(v) || 0.0
}

console.log(floatval("122.34343The"))
console.log(floatval("+122.34343.1"))
console.log(floatval("-122.34343.1"))
console.log(floatval("-.34343.1"))
console.log(floatval("The122.34343"))
